ui/src/bibliographyGenerator/__tests__/BibliograpyGeneratorPageContainer.test.jsx

```diff
@@ -17,7 +21,18 @@ describe('BibliographyGeneratorPageContainer', () => {
     jest.clearAllMocks();
   });
 
-  it('passes props down', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('passes props to BibliographyGenerator component', () => {
+    const expectedState = {
+      data: fromJS({ download_url: 'https://google.com' }),
+      citationErrors: fromJS([{ message: 'Citation error 1' }]),
+      error: fromJS({ message: 'Error' }),
+      loading: false,
+    };
+
     const store = getStoreWithState({
```

Comment:
Since you are touching the files, replace this with `getStore`as `getStoreWithState` is deprecated and will be removed.